bootcamp-milestone-2/src/app/resume/page.tsx

```diff
@@ -0,0 +1,90 @@
+import style from "./page.module.css";
+
+
+export default function Resume() {
+    return (
+        <div className={style.joe}>
+            <main>
+                <h1 className="page-title">Resume</h1>
+                <br />
+                <a href="PlaceholderPage.pdf" download className="text-white"> Download Resume </a>
+
+                <div className="resume">
+                    <section className="section">
+                        <h2 className="section-title">
+                            Education
+                        </h2>
+                        <div className="entry">
+                            <h3 className="entry-title">California Polytechnic State University, San Luis Obispo</h3>
+                            <p className="entry-info">Bachelor of Science in Computer Science - Expected Graduation: June 2026</p>
+                            <p className="entry-description"></p>
+                        </div>
+                    </section>
+                    <section className="section">
+                        <h2 className="section-title">
+                            Experience
+                        </h2>
+                        <div className="entry">
+                            <h3 className="entry-title">Software Developer Intern</h3>
+                            <p className="entry-info">Diaryz | June 2023 – Present</p>
+                            <p className="entry-description">Developed RESTful APIs using Django’s Rest Framework to communicate
+                                with a React Native app</p>
+                        </div>
+                        <div className="entry">
```

Comment:
I would consider creating a component for entries on your resume and using an array of entries to pass data to it like you did with Blogs